terraform/environments/mlra/main.tf

```diff
@@ -99,6 +99,26 @@ resource "aws_lb_target_group" "alb_target_group" {
 #  appnameenv = "${local.application_name}-${local.environment}"
 #}
 
+# Get the map of pagerduty integration keys from the modernisation platform account
+# SNS topic for monitoring to send alarms to
+resource "aws_sns_topic" "high_priority" {
+  name = "high_priority"
+}
+
+data "aws_secretsmanager_secret" "pagerduty_integration_keys" {
+  provider = aws.modernisation-platform
+  name     = "pagerduty_integration_keys"
+}
 
+data "aws_secretsmanager_secret_version" "pagerduty_integration_keys" {
+  provider  = aws.modernisation-platform
+  secret_id = data.aws_secretsmanager_secret.pagerduty_integration_keys.id
+}
+
+module "pagerduty_core_alerts" {
+  source                    = "github.com/ministryofjustice/modernisation-platform-terraform-pagerduty-integration?ref=1.0"
```

Comment:
I do not think the source ref here would work (despite what's stated in the module doc), since the tag they used in the module repo is [v1.0.0](https://github.com/ministryofjustice/modernisation-platform-terraform-pagerduty-integration/releases/tag/v1.0.0), so needs changing the suffix of this link to match the tag

Reply:
version number changed